<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_invest</class>
 <widget class="QDialog" name="Dialog_invest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:36px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Banca
Transilvania

Company 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>ALRO

Company 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>390</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>BRD

Company 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>460</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>61</y>
     <width>551</width>
     <height>151</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
background-color: rgb(229, 229, 229);</string>
   </property>
   <property name="text">
    <string>Choose a company to visit their webpage and development in time.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1211</width>
     <height>701</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="Resurse.qrc">:/fundalPg/BANNER.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>pushButton</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources>
  <include location="Resurse.qrc"/>
 </resources>
 <connections/>
</ui>
